video/external/src/backend.rs

```diff
@@ -49,6 +50,11 @@ impl ExternalVideoBackend {
         const OS: &str = std::env::consts::OS;
         const ARCH: &str = std::env::consts::ARCH;
 
+        let local_filenames = match OS {
+            "linux" => vec!["libopenh264.so.7", "libopenh264.so.2.4.1", "libopenh264.so"],
+            _ => vec![],
```

Comment:
I get that it's far less likely to have this library already installed on any other OS - but do we just assume that it's basically impossible, or should at least a `// TODO: investigate other OSes` comment be left here? I don't know, just wondering...

Reply:
Of course, the reason I added a `match` here was to make it easy to add support for other OSes, it's just that I have no idea what names I should use or how OpenH264 is installed there.

(Added the TODO)